hier_config/hc_child.py

```diff
@@ -431,7 +427,7 @@ def set_order_weight(self):
         """
 
         for child in self.all_children():
-            for rule in self.options['ordering']:
+            for rule in self.root.options['ordering']:
```

Comment:
with `self.options` property created, we can change this back to `self.options...`